src/Presentation/Console/Commands/InitProjectCommand.php

```diff
@@ -180,11 +180,18 @@ protected function initializeSettingValues(array $options, array $settingEntitie
                 );
             }
 
-            $values = match ($settingEntity->getType()) {
-                'boolean' => (bool)$values,
-                'array' => (array)$values,
-                default => (string)$values,
-            };
+            switch ($settingEntity->getType()) {
```

Comment:
Please rename var to `$value` because it is exactly one value.
Also, please replace the switch clause with an extra method `typeValue($value)`. Use early return in it.